apps/keyboard/test/unit/keyboard/layout_rendering_manager_test.js

```diff
@@ -57,7 +57,7 @@ suite('LayoutRenderingManager', function() {
       viewManager: viewManager
     };
 
-    Object.defineProperty(window, 'hidden', { value: false });
+    Object.defineProperty(window, 'hidden', { value: false, configurable: true });
```

Comment:
This won't pass the lint checking for 80 chars a line, please put a newline after `{`.

Thanks.